test/speed.cpp

```diff
@@ -28,7 +28,7 @@ SCENARIO("container resolves automatic values quickly", "[performance]") {
         container.bind<symbols::symbolC>().to<ServiceC>();
 
         WHEN("the dependency is resolved") {
-            const int iterations = 100000000;
```

Comment:
Btw, Catch2 supports benchmarking natively. I have not tried it yet, but if you run your test binary with `--help` you'll see options for benchmarking.